### ISSUE TITLE
Weekly Update - Freddie

### ISSUE THREAD

**Issue by FreddieIreland** [FreddieIreland/cvxctfwebsite#2]

**Comment by FreddieIreland** [comment]
1. This past week, I've been working on setting up the kanban board and thinking about things that needed to be added to the "to do section". With the help of Rowan, together we wrote up and edited an email which we agreed on sending out to Coach Rob tomorrow as he had a super busy week last week with XC league finals and teaching. 
2. This week, the only issue that was fixed was a small problem I had with cloud9, which was everytime that I opened up cloud9 every single file would automatically open up and it would be a pain to close them all and find exactly while folder I wanted to work on.
3. The issues that I'm currently facing revolve around github and pushing our code out to the repository. When you push from cloud9 to github, it asks for a Personal Access Token which seems to not work sometimes and always expire. Therefore, Rowan may have to use my token or vice versa, showing that something I worked on may have been pushed by Rowan which can be a bit confusing.
4. In the future, a problem that I can see happening is what exactly happens when there is new information to be added to the site as our XC & TF season move on. I'm not sure if me and Rowan will have to add that information or if there's some way to make a panel where the coaches can easily access and upload this new information.
5. As of now, I'm not stuck on anything since we're not taking any huge steps of making the site as we want to wait to get Coach Rob's reply and both of us are finishing up the HTML & CSS course. 
6. This upcoming week, I'm going to be working on watching the remainder of the HTML & CSS course, collaborating with Rowan to give a layout to our website and finally host our website through github.
7. For this upcoming week, I'm really excited to finish up the HTML & CSS course and have a complete note sheet which I can reference to whenever I need. I'm also looking forward to hearing back from Coach Rob and seeing what he thinks is best when it comes to how we should design the website. I also am looking forward to finishing up the general layout of our website once we hear back from Coach Rob as it will be huge progress.

**Comment by FreddieIreland** [comment]
1. This past week, I continued to work on the HTML & CSS course, closing in on nearly half way through with it. I also got a reply from Coach Rob, which he was thrilled to hear our idea and we are soon going to be having weekly meetings with him to discuss progress. 
2. An issue that I fixed this week was with personal access tokens, which was a problem from last week. Both Rowan and I now have our individual personal access tokens to push and pull from GitHub through cloud9.
3. An issue that I'm currently facing is with my Chromebook. Every time I go back and forth in-between coding through cloud9 and then tabbing back to the HTML & CSS course, the screen becomes completely white and takes a good 10-15 seconds to load whatever is on the tab I'm trying to switch to. My Chromebook has been extremely slow this entire semester as it's an older one, but it's not that big of an issue which is preventing me from getting my work done.
4. In the future, another problem that popped into my head was the conversion of the website from XC to Track season. I'm not entirely sure if we'll completely change the layout, but most likely the general format of the website will stay the same.
5. Again, I'm not stuck on anything and will always look out for help whenever I come across something that both Rowan and I cannot fix.
6. This upcoming week, my goals are very similar to last weeks, the only addition being coming up with a day to meet with coach every week.
7. I'm super excited for this week and Thanksgiving break, mainly looking forward to finishing my course and continuing to see the results of the website!

**Comment by FreddieIreland** [comment]
Freddie Ireland - Week #3
1. This past week, I was given the opportunity to create C++ videos for the buffets that covers a wide range of the curriculum. I am now a collaborator of the NeatPoole22 textbook and I am able to push my videos into the textbook whenever they've been created. I also am still working through the HTML & CSS course, and the notes file is getting more compact and detailed by the minute. 
2. An issue that was fixed this week was my personal understanding of semantic elements in HTML. These are used to clearly let the browser know what the page is about, allowing for better search engine compatibility.
3. An issue that I am currently facing is sometimes things were not properly saved on AWS, and everytime I come back to my notes I have to go to the history of the file and copy and paste the most updated version of what I have written down. 
4. A problem that I may see in the future has to do with organization of my notes, as I would like it to be one area of all the HTML tags, HTML attributes, CSS styles, etc. However, at the moment everything is just in the order from which I learned it, meaning the organization is not the best. 
5. Again, I am not stuck on anything and I only really need help on one small formatting thing with AWS. 
6. For this upcoming week, I'm going to be refreshing myself on .h files and methods for C++ and create a video for that. I'm also finished up with the HTML side of the course and going to start moving into the CSS, which includes all the essentials for styling. 
7. Something I am excited for this upcoming week is talking with Coach Rob as he is the "customer" for our website. I want to hear all his thoughts and design the website to his liking.
